src/commands/code-analyzer/rules.ts

```diff
@@ -1,19 +1,40 @@
-import {SfCommand} from '@salesforce/sf-plugins-core';
+import {Flags, SfCommand} from '@salesforce/sf-plugins-core';
+import {RulesAction} from '../../lib/actions/RulesAction';
+import {BundleName, getMessage} from '../../lib/messages';
 
-export default class Rules extends SfCommand<void> {
+export default class RulesCommand extends SfCommand<void> {
 	// We don't need the `--json` output for this command.
 	public static readonly enableJsonFlag = false;
-	public static readonly summary = 'summary boop'; // TODO
-	public static readonly description = 'description boop'; // TODO
-	public static readonly examples = [ // TODO
-		'example1 boop',
-		'example2 boop'
+	public static readonly summary = getMessage(BundleName.RulesCommand, 'command.summary');
+	public static readonly description = getMessage(BundleName.RulesCommand, 'command.description');
+	public static readonly examples = [
+		getMessage(BundleName.RulesCommand, 'command.examples')
 	];
 
-	public static readonly flags = {};
+	public static readonly flags = {
+		'rule-selector': Flags.string({
+			summary: getMessage(BundleName.RulesCommand, 'flags.rule-selector.summary'),
+			char: 'r',
+			multiple: true,
+			delimiter: ',',
+			default: ["recommended"]
```

Comment:
Question... not that it matters because it will be case insensitive in matching... but do think we'll want tags to be camelCase, PascalCase, lowercase, or snake-case?

I assumed we'd probably use PascalCase for the tags... thus I have "Recommended" hard coded in the core module. But I can go either way.

I ask because I believe this default value shows up not only in the generated help text... but will show up in the generated help doc that gets published.

Reply:
My personal preference is probably kebab-cased, but I think I'm also okay with PascalCased. I can change it to "Recommended" if need be. Have we asked PM what their preference is?